sources/packages/forms/src/form-definitions/sfaa2022-23.json

```diff
@@ -8576,8 +8575,8 @@
                       "shortcut": ""
                     },
                     {
-                      "label": "you will have finished 4 years of full-time, postsecondary study in B.C. before the first day of your study period.",
-                      "value": "youWillHaveFinished4YearsOfFullTimePostsecondaryStudyInBCBeforeTheFirstDayOfYourStudyPeriod",
```

Comment:
I thought you could choose a shorter name for this but I see the other values are also very long names. Maybe not for this PR.